docs/installation/server-config/security.rst

```diff
@@ -123,30 +125,32 @@ with |c3-short|.
 Configuring Authorization of KSQL with Kafka ACLs
 -------------------------------------------------
 
-You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the KSQL cluster is
-interactive or non-interactive.
+Kafka clusters can use ACLs to control access to resources. Such clusters require each clients to authenticate as a particular user.
+To work with such clusters, KSQL must be configured to :ref:`authenticate with the Kafka cluster <config-security-ssl-sasl>`,
+and certain ACLs must be defined in the Kafka cluster to allow the user KSQL is authenticating as access to resources.
+The list of ACLs that must be defined depends on whether the KSQL cluster is interactive or non-interactive, and are covered below.
```

Comment:
"the KSQL cluster is interactive or non-interactive" -> "the KSQL cluster is configured for [interactive or non-interactive (headless) use](internal link to relevant section under Configuring KSQL)"

Reply:
Done